arch/lkl/kernel/vmlinux-fuzzing-postlink.lds

```diff
@@ -0,0 +1,40 @@
+OUTPUT_FORMAT("elf64-x86-64")
```

Comment:
I don't think this file is needed, it should be automatically generated from the lds.S file, right?

Reply:
Ah, yes, indeed it will be generated from vmlinux-fuzzing-postlink.lds.S. Many thanks for catching this -- looks like I mistakenly included the generated file in the commit. Removing it.